ansible/accumulo.yml

```diff
@@ -37,14 +37,14 @@
 - hosts: workers
   tasks:
     - name: "start accumulo 2.0 tablet servers"
-      command: "{{ accumulo_home }}/bin/accumulo-service tserver start"
+      command: "nohup {{ accumulo_home }}/bin/accumulo-service tserver start"
       register: start_result
       changed_when: "'Starting' in start_result.stdout"
       when: accumulo_major_version == '2'
 - hosts: accumulomaster
   tasks:
     - name: "start accumulo 2.0 master, monitor, gc & tracer"
-      command: "{{ accumulo_home }}/bin/accumulo-service {{ item }} start"
+      command: "nohup {{ accumulo_home }}/bin/accumulo-service {{ item }} start"
       register: start_result
       changed_when: "'Starting' in start_result.stdout"
```

Comment:
I wonder if ansible thinks the command is done when it sees this in the output and that is why the signal is sent before the command is done.

Reply:
I think you're right about that. The "Starting" message occurs prior to actually executing the java process (which is already executed with `nohup`).